scripts/package-suc

```diff
@@ -8,6 +8,7 @@ cd $(dirname $0)/..
 mkdir -p dist/artifacts
 cp bin/rancher-system-agent dist/artifacts/rancher-system-agent${SUFFIX}
 cp install.sh dist/artifacts/install.sh
+cp uninstall.sh dist/artifacts/uninstall.sh
```

Comment:
i think you need a line in the dockerfile too: https://github.com/rancher/system-agent/blob/main/package/Dockerfile.suc#L13 the drone job is just the docker plugin pointing to the .suc dockerfile: https://github.com/rancher/system-agent/blob/main/.drone.yml#L59 and this looks like a helper `make package-suc` for local dev